src/ethlance/server/core.cljs

```diff
@@ -88,6 +88,8 @@
                  :write-events-into-file? true
                  :file-path "ethlance-events.log"}
 
+   :event-store {:store-file "event-store.log"}
```

Comment:
Here you refer to it as :event-store, but namespace is `ethlance.server.events-store` (eventS). Please make ti consistent as event-store everywhere

Reply:
Done!

Reply:
hmm but I wrote make it as event-store, not events-store

Reply:
Meh sorry, read it the other way around, changed everything to event-store